OracleAccessManagement/dockerfiles/12.2.1.4.0/Dockerfile

```diff
@@ -0,0 +1,176 @@
+    CONTAINER_DIR=/u01/oracle/user_projects/container \
+    ADMIN_LISTEN_HOST="${ADMIN_LISTEN_HOST:-}" \
+    ADMIN_NAME="${ADMIN_NAME:-AdminServer}" \
+    ADMIN_LISTEN_PORT="${ADMIN_LISTEN_PORT:-7001}" \
+    DOMAIN_TYPE="${DOMAIN_TYPE:-oam}" \
+    RCUPREFIX=${RCUPREFIX:-OAM01} \
+    DB_USER=${DB_USER:-} \
+    DB_PASSWORD=${DB_PASSWORD:-} \
+    DB_SCHEMA_PASSWORD=${DB_SCHEMA_PASSWORD:-} \
+    USER_MEM_ARGS=${USER_MEM_ARGS:-"-Djava.security.egd=file:/dev/./urandom"} \
+    JAVA_OPTIONS="${JAVA_OPTIONS} -Dcom.sun.jndi.ldap.object.disableEndpointIdentification=true" \
+    PATH=$PATH:/usr/java/default/bin:$ORACLE_HOME/oracle_common/common/bin:/u01/oracle/wlserver/common/bin:/u01/oracle/dockertools
+
+#
+# Creation of User, Directories and Installation of OS packages
+# ----------------------------------------------------------------
+USER root
+RUN mkdir -p ${BASE_DIR} && \
+#    useradd -b ${BASE_DIR} -d ${ORACLE_HOME} -m -s /bin/bash oracle && \
+    chmod a+xr ${BASE_DIR} && chown oracle:oracle ${BASE_DIR} && \
+    mkdir -p ${USER_PROJECTS_DIR} && \
+    chown -R oracle:oracle ${USER_PROJECTS_DIR} && chmod -R 775 ${USER_PROJECTS_DIR} && \
+    mkdir -p ${CONTAINER_DIR} && \
+    chown -R oracle:oracle ${CONTAINER_DIR} && chmod -R 775 ${CONTAINER_DIR} && \
+    mkdir -p ${SCRIPT_DIR} && chown oracle:oracle ${SCRIPT_DIR} && \
+    mkdir -p ${PROPS_DIR} && chown oracle:oracle ${PROPS_DIR} && \ 
+    yum install -y libaio && \
+    yum install -y hostname && \
+    rm -rf /var/cache/yum && \
+    mkdir ${PATCH_DIR} && \
+    mkdir ${OPATCH_PATCH_DIR} && \
+    chown -R oracle:oracle ${BASE_DIR} && \
+    chown -R oracle:oracle ${PATCH_DIR} && \
+    chown -R oracle:oracle ${OPATCH_PATCH_DIR}
+
+#
+FROM base as builder
+# Copy packages and scripts
+# -------------------------
+COPY --chown=oracle:oracle Dockerfile patches/* ${PATCH_DIR}/
+COPY --chown=oracle:oracle Dockerfile opatch_patch/* ${OPATCH_PATCH_DIR}/  
+COPY container-scripts/* ${SCRIPT_DIR}/
+COPY install/* ${BASE_DIR}/
+ADD  $FMW_IDM_JAR ${BASE_DIR}/
+
+#
+# Update Permissions for packages and scripts
+# --------------------------------------------
+RUN cd ${BASE_DIR} && chmod 755 *.jar && \
+     chmod a+xr ${SCRIPT_DIR}/* && \
+     chown -R oracle:oracle ${CONTAINER_DIR} && chmod -R 775 ${CONTAINER_DIR} && \
+     chown oracle:oracle ${SCRIPT_DIR}/*
+
+# 
+# Installation of IDM Binaries
+# --------------------------------------------
+USER oracle
+WORKDIR ${ORACLE_HOME}
+RUN cd ${BASE_DIR} && \
+#install IDM in silent mode
+ $JAVA_HOME/bin/java -jar ${BASE_DIR}/$FMW_IDM_JAR -silent -responseFile ${BASE_DIR}/iam.response -invPtrLoc ${ORACLE_HOME}/oraInst.loc -jreLoc $JAVA_HOME -ignoreSysPrereqs -force -novalidation ORACLE_HOME=${ORACLE_HOME} && \
+ rm -fr ${BASE_DIR}/*.jar ${BASE_DIR}/*.response && \
+ rm -f ${OPATCH_PATCH_DIR}/Dockerfile && \
+ rm -f ${PATCH_DIR}/Dockerfile
+
+#
+# Apply patch to OPatch
+#
+USER oracle
+WORKDIR ${ORACLE_HOME} 
```

Comment:
The `WORKDIR` is already `${ORACLE_HOME}` (set on line 101). This directive isn't needed.